hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/index/SqlIndexTest.java

```diff
@@ -28,7 +28,7 @@
 
 @RunWith(HazelcastParametrizedRunner.class)
 @Parameterized.UseParametersRunnerFactory(HazelcastSerialParametersRunnerFactory.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
+@Category({SlowTest.class, ParallelJVMTest.class})
```

Comment:
This should be an exception, because it tests the _basic_ index scan logic.

Reply:
da66388a2fee9a9f64ab724e34f574aef33cd6b1